### PR TITLE
feat(input-editable): variation of font and width

### PR DESCRIPTION
**Pull request by pcoliveira90**
It was implemented the possibility to define the font and width based on the pre-defined sizes:
 Font-size:  'fs-10' | 'fs-12' | 'fs-14' | 'fs-16' | 'fs-20' | 'fs-24' | 'fs-32' | 'fs-40';
 Width : 'exp-0' | 'exp-10' | 'exp-20' | 'exp-30' | 'exp-40' | 'exp-50' | 'exp-60' | 'exp-70' | 'exp-80' | 'exp-90' | 'exp-100';

### CODE REVIEW

**Review by WillianLomeu — CHANGES_REQUESTED** [review]
I can't approve this changes without a request form.

**Comment by vinioo** [comment]
I think maybe its better to customize the component as you need on your project. If you aren't able to customize it due to the shadow DOM, maybe you can add some **part** attributes to the input-editable and then style it using the ```**:part()**``` selector in your project.

**Comment by pcoliveira90** [comment]
An alignment was made with Balbo and Tathi. It was defined the possibility to expand from 0 to 100 and sources from 16 to 40.

File: src/index.html [hunk 1]
```diff
@@ -14,8 +14,44 @@
     }
   </style>
 
-  <body>
+  <body >
```
Comment (WillianLomeu) [inline src/index.html:17]: Please, remove this content of the body.

File: src/globals/_expands.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*
+* Percentages for component expansion
+* For percentage 0, a default value of 140px was defined
+*/
+
+$exp_0: 140px;
```
Comment (WillianLomeu) [inline src/globals/_expands.scss:6]: Please, remove this variables. The width can be the minor (140px default) and bigger with 100%.

File: src/components/input-editable/input-editable.tsx [hunk 3]
```diff
@@ -103,50 +117,50 @@ export class InputEditable {
...
+          <div class={{ 'input__editable--static': true, 
+                        'input__editable--hidden': this.isEditing }}
+               onClick={this.handleEditing}
           >
             <bds-typo
               tag="span"
               part="input__editable--static__typo"
               class="input__editable--static__typo"
-              variant="fs-24"
+              variant={this.variant}
```
Comment (WillianLomeu) [inline src/components/input-editable/input-editable.tsx:129]: These font variants are just two. Default (24) and for 100% width (16). Can't be chanceable.

File: src/components/input-editable/input-editable.tsx [hunk 4]
```diff
@@ -13,6 +15,18 @@ export interface InputEditableEventDetail {
...
 
+   /**
+   * Variant. Entered as one of the font size variant. Can be one of:
+   * 'fs-16' | 'fs-20' | 'fs-24' | 'fs-32' | 'fs-40';
+   */
+  @Prop() variant?: FontSizeInputEditable = 'fs-24';
+
+  /**
+   * Defines whether the input will expand.
+   * 'exp-0' | 'exp-10' | 'exp-20' | 'exp-30' | 'exp-40' | 'exp-50' | 'exp-60' | 'exp-70' | 'exp-80' | 'exp-90' | 'exp-100';
```
Comment (WillianLomeu) [inline src/components/input-editable/input-editable.tsx:26]: Change these options. Just boolean (dafault or expansible).

File: src/components/input-editable/input-editable.tsx [hunk 5]
```diff
@@ -13,6 +15,18 @@ export interface InputEditableEventDetail {
 export class InputEditable {
   @Element() el!: HTMLBdsInputEditableElement;
 
+   /**
+   * Variant. Entered as one of the font size variant. Can be one of:
+   * 'fs-16' | 'fs-20' | 'fs-24' | 'fs-32' | 'fs-40';
```
Comment (WillianLomeu) [inline src/components/input-editable/input-editable.tsx:20]: Remove theses options. Fonts are fixed acord to width.

File: src/components/input-editable/input-editable.scss [hunk 6]
```diff
@@ -1,5 +1,65 @@
...
+    @include part_input_font_size($fs-24);
+  }
+  &.fs-32::part(input){
+    @include part_input_font_size($fs-32);
+  }
+  &.fs-40::part(input){
+    @include part_input_font_size($fs-40);
+  }
+}
+@mixin input_max_width() {
```
Comment (WillianLomeu) [inline src/components/input-editable/input-editable.scss:25]: These widths are just two. Default (140px) and for 100%. Can't be chanceable.

**Review by WillianLomeu — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (pcoliveira90) [reply src/index.html:17]: resolved

**Review by WillianLomeu — APPROVED** [review]

**Pull request merged**